.github/workflows/build.yml

```
@@ -1,8 +1,24 @@
 name: Compile & Test
 
-on: [push]
-
+on: [push, pull_request]
 jobs:
+  codegen-verify:
+    name: Verify code generation
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+
+      - uses: actions/checkout@v1
+
+      - name: install mockgen
+        run: go get github.com/golang/mock/mockgen@v1.4.3
```

This fails because `go get` does not install mockgen into `$GOPATH`, but as a dependency of the current gomodule. Maybe...

```suggestion
        run: |
          cd /
          go get github.com/golang/mock/mockgen@v1.4.3
          cd -
```